core-services/release-controller/_releases/release-ocp-4.7-ci.json

```diff
@@ -119,7 +119,7 @@
       "cron":"0 3,16,22 * * *",
       "upgrade":true,
       "upgradeFrom": "PreviousMinor",
-      "prowJob":{"name":"release-openshift-ocp-installer-e2e-ovirt-upgrade-4.6-stable-to-4.7-ci"}
+      "prowJob":{"name":"periodic-ci-openshift-release-master-ocp-4.7-e2e-ovirt-upgrade"}
```

Comment:
Why the rename? `ovirt-upgrade` should be generic upgrade job, `ovirt-upgrade-4.6-stable-to-4.7-ci` is specific to stable-to-nightly upgrade.

Reply:
I'll change that

Reply:
So I tried to change it but got an error from the config saying:
```
"e2e-ovirt-upgrade-4.6-stable-to-4.7-ci": a lowercase RFC 1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```
So I think he is not a fan of the '.' in the name :/
We can create a separate variant for ocp upgrades (like there is for origin), but till that is done I think I'll keep the "e2e-ovirt-upgrade" name since it is clear to me that a 4.7 upgrade job upgraded from 4.6

Reply:
This works perfectly in other places:
```
"prowJob":{"name":"release-openshift-origin-installer-e2e-aws-4.1"}
```